docs/api-ref.md

```diff
@@ -2709,6 +2709,215 @@ The `SubscriptionItem`.  See [SubscriptionItem class](#subscriptionitem-class)
 <br>
 <br>
 
+---
+
+## Tasks
+
+Using the TSC library, you can get information about all the tasks on a site and you can remove tasks. To create new tasks see [Schedules](#schedules).
+
+The task resources for Tableau Server are defined in the `TaskItem` class. The class corresponds to the task resources you can access using the Tableau Server REST API. The task methods are based upon the endpoints for tasks in the REST API and operate on the `TaskItem` class.  
+
+### TaskItem class
+
+```py
+TaskItem(id, task_type, priority, consecutive_failed_count=0, schedule_id=None, target=None)
+```
+
+**Attributes**
+
+Name | Description  
+:--- | :---  
+`consecutive_failed_count` | The number of failed consecutive executions.  
+`id` |   The id of the task on the site.  
+`priority` | The priority of the task on the server.
+`schedule_id` | The id of the schedule on the site.    
+`target` | An object, `datasource` or `workbook` which is associated to the task. Source file: models/target.py
+`task_type` | Type of extract task - full or incremental refresh.  
```

Comment:
Suggest reordering these to be in the order shown in TaskItem class above, so id, task_type, ...).